flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImplStartStrategy.java

```diff
@@ -79,8 +79,8 @@ public void testTableScanThenIncrementalStrategy() throws IOException {
             .startingStrategy(StreamingStartingStrategy.TABLE_SCAN_THEN_INCREMENTAL)
             .build();
 
-    // empty table
     assertThat(ContinuousSplitPlannerImpl.startSnapshot(TABLE_RESOURCE.table(), scanContext))
+        .as("empty table")
```

Comment:
Along with the backport, set each `empty table` comment in this class to `as` for refactoring tests.